messages/whitespace-found.txt

```diff
@@ -0,0 +1,3 @@
+ESLint couldn't find the plugin "eslint-plugin-<%- pluginName %>". It looks like there is whitespace in the plugin name.
```

Comment:
Now that the `longName` is being used in the `messageData`, I think this should just be `ESLint couldn't find the plugin "<%- longName %>".` @mysticatea can you confirm as well please?